language/diem-framework/modules/NetworkIdentity.move

```diff
@@ -44,93 +46,91 @@ module DiemFramework::NetworkIdentity {
     }
     spec initialize_network_identity {
         let account_addr = Signer::spec_address_of(account);
-        ensures exists<NetworkIdentity>(account_addr);
         modifies global<NetworkIdentity>(account_addr);
     }
 
     /// Return the underlying `NetworkIdentity` bytes
     public fun get(account_addr: address): vector<vector<u8>> acquires NetworkIdentity {
-        assert(exists<NetworkIdentity>(account_addr), ENETWORK_ID_DOESNT_EXIST);
+        assert(
+            exists<NetworkIdentity>(account_addr),
+            Errors::invalid_state(ENETWORK_ID_DOESNT_EXIST)
```

Comment:
I think we generally use Errors::not_published or similar, but they can be also use cases where invalid state is more appropriate.